recipes-browser/cog/cog/0001-cmake-Fix-CMake-v3.16-cannot-create-ALIAS-target-Wpe.patch

```diff
@@ -0,0 +1,43 @@
+From eddc6a58f84d4b89599101ff374290fba7e7645c Mon Sep 17 00:00:00 2001
+From: Pablo Saavedra <psaavedra@igalia.com>
+Date: Tue, 6 Sep 2022 11:24:07 +0200
+Subject: [PATCH] cmake: Fix CMake (v3.16) cannot create ALIAS target
+ "Wpe::FDO" error
```

Comment:
You are adding this patch here, but then it is not used in any recipe. Moreover, version 0.14.1 already includes this fix. It should be possible to remove the file from this PR.

Reply:
Good catch.